index.html

```diff
@@ -40,7 +40,7 @@ <h1>Mohamed Shaaban</h1>
         <p class="title">Web Engineer</p>
         <ul class="contacts fadeIn">
           <li>
-            <a target="_blank" rel="noopener noreferrer" href="/resume/">
+            <a target="_self" rel="noopener noreferrer" href="/resume/">
```

Comment:
Agreed
Can we drop `target` attribute all together as `_self` is the default browser target if not specified?

Reply:
sure we can but i left it as it's available in the rest of tags so just as we follow the same convention as other tags.

Reply:
I guess it is safe to say, we can drop all `target` from all links

My first intention was to let the viewer navigate through all social/professional links without the need to use browser’s back button, but it proved to be not user friendly approach. WDYT?

Reply:
in the social media links it's reasonable to do so since these are external sites. with CV/Resume it's not reasonable since it's the same site.

Reply:
Convinced